mailpoet/lib/AdminPages/Pages/Automation.php

```diff
@@ -36,6 +42,7 @@ public function render() {
         'root' => rtrim($this->wp->escUrlRaw($this->wp->restUrl()), '/'),
         'nonce' => $this->wp->wpCreateNonce('wp_rest'),
       ],
+      'workflowCount' => count($this->workflowStorage->getWorkflows()),
```

Comment:
I thought it would be worthwhile to have the count immediately on page load so we don't need a "loading" state before deciding to render either the onboarding component or the listing itself.  If that seems like a worthy cause I would add a method to workflowStorage to retrieve a count directly from the DB instead of loading them all into memory like this.

Reply:
@johnolek That's good. We can also consider inlining ("preloading") the first page here fully but this is fine now.